[llvm-c-test] Rename --test-dibuilder-debuginfo-format to `--test-dibuilder`

vadorovsky opened the pull request:
The former name was introduced during the split between debug info intrinsic and `DbgRecord`. Before the split, it was named `--test-dibuilder`.

However, the full migration to `DbgRecord` happened already and we have just one test suite related to building debug info using LLVM-C API. Therefore, it makes sense to rename it back to `--test-dibuilder`.

weliveindetail reviewed (approved):
Good catch! LGTM

vadorovsky commented:
@weliveindetail @OCHyams do you mind merging this one as well?

OCHyams reviewed (commented):
Thanks, good spot.

Looking at it though, I think the original name `--test-dibuilder` from before all the renaming makes the most sense in today's state. I think we probably should've restored that name in the patch you linked - I probably just missed that at the time.

Could I be cheeky and ask that you invert your patch; change the flag name to `--test-dibuilder` (keep the existing usage message) and update `debug_info_new_format.ll` to use that flag?

vadorovsky committed:
[llvm-c-test] Rename `--test-dibuilder-debuginfo-format` to `--test-dibuilder`

The former name was introduced during the split between debug info
intrinsic and `DbgRecord`. Before the split, it was named
`--test-dibuilder`.

However, the full migration to `DbgRecord` happened already and we have
just one test suite related to building debug info using LLVM-C API.
Therefore, it makes sense to rename it back to `--test-dibuilder`.

vadorovsky commented:
@OCHyams done

OCHyams reviewed (approved):
Thanks!
